src/main/scala/se/kth/cda/arc/ast/printer/MLIRPrinter.scala

```diff
@@ -88,6 +88,20 @@ object MLIRPrinter {
       tmp
     }
 
+    def dumpVec(identifiers: SymbolMap, elems: Vector[Expr], ty: Type): String = {
+      val elemTy = ty match {
+        case Vec(et) => et
+        case x       => x
+      }
+      val es = elems.map(e => e.toMLIR(identifiers))
```

Comment:
```scala
elems.map(e => e.toMLIR(identifiers))
```
can be written as:
```scala
elems.map(_.toMLIR(identifiers))
```
It's not exactly important, just stylistic 😅

Reply:
This is the kind of tips I appreciate as I can't say that I really know Scala.